generators/app/templates/src/app/screens/Auth/screens/Login/index.test.js

```diff
@@ -40,15 +40,9 @@ describe('<Login />', () => {
     const emailInput = getByTestId('Email');
     const passwordInput = getByTestId('Contraseña');
     const submitButton = getByText('Iniciar sesión');
-    await waitFor(async () => {
-      await fireEvent.changeText(emailInput, VALID_EMAIL);
-    });
-    await waitFor(async () => {
-      await fireEvent.changeText(passwordInput, VALID_PASSWORD);
-    });
-    await waitFor(async () => {
-      await fireEvent.press(submitButton);
-    });
+    fireEvent.changeText(emailInput, VALID_EMAIL);
+    fireEvent.changeText(passwordInput, VALID_PASSWORD);
+    await waitFor(() => fireEvent.press(submitButton));
```

Comment:
Did you check if you don't put a valid email and password that submit is not been called? `expect(dispatch).toHaveBeenCalledTimes(0);`

Reply:
Good idea! I added a new expect